symforce/codegen/codegen.py

```diff
@@ -828,6 +838,7 @@ def with_jacobians(
         include_results: bool = True,
         name: str = None,
         sparse_jacobians: bool = False,
+        output_symbols: T.Optional[T.Set[sf.Symbol]] = None,
```

Comment:
I think this one still needs to be forwarded to the `Codegen` constructor below?